packages/rich-text/setup/index.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * WordPress dependencies
+ */
+import { render, createElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { RichText, toggleFormat, insert } from '../src';
+
+// Expose component to be tested.
+window.test = { render, createElement, RichText, toggleFormat, insert };
```

Comment:
Shouldn't we follow what you see in WordPress to make it easier to switch?
```suggestion
window.wp.RichText = { render, createElement, RichText, toggleFormat, insert };
```

Reply:
What do you mean with switching?

Reply:
Switch context, in WordPress you have RichText exposed under `wp.RichText` global.

Reply:
I figured this was fine, since this is a test environment, not a WordPress environment, where we won't have everything exposed, only the necessary functions to test the component.

Reply:
It would work as long as you don't need to expose anything from other packages. We can start with `test` and refactor later if it turns out to be necassary.